src/answers-umd.js

```diff
@@ -19,22 +19,20 @@ import { AnswersComponentError } from './core/errors/errors';
 import AnalyticsEvent from './core/analytics/analyticsevent';
 import StorageKeys from './core/storage/storagekeys';
 import QueryTriggers from './core/models/querytriggers';
-import SearchConfig from './core/models/searchconfig';
 import AutoCompleteApi from './core/search/autocompleteapi';
 import MockAutoCompleteService from './core/search/mockautocompleteservice';
 import QuestionAnswerApi from './core/search/questionanswerapi';
 import MockQuestionAnswerService from './core/search/mockquestionanswerservice';
 import SearchApi from './core/search/searchapi';
 import MockSearchService from './core/search/mocksearchservice';
 import ComponentManager from './ui/components/componentmanager';
-import VerticalPagesConfig from './core/models/verticalpagesconfig';
-import { SANDBOX, PRODUCTION, LOCALE, QUERY_SOURCE } from './core/constants';
 import MasterSwitchApi from './core/utils/masterswitchapi';
 import RichTextFormatter from './core/utils/richtextformatter';
 import { isValidContext } from './core/utils/apicontext';
 import FilterNodeFactory from './core/filters/filternodefactory';
 import { urlWithoutQueryParamsAndHash } from './core/utils/urlutils';
 import TranslationProcessor from './core/i18n/translationprocessor';
+import AnswersConfig from './core/models/answersconfig';
```

Comment:
Should I add a typeDef here?

Reply:
I believe that @oshi97 likes to add them so that the IDE can read the jsdocs from the class. Are you able to see any help text when you click on the class name?

Reply:
Yeah I like my intellisense haha, it makes me feel like I'm doing things correctly. If your favorite code completion is working properly already there's no need.

Reply:
Since we're using the AnswersConfig constructor directly here I think we should be fine